### PR TITLE
Tailwind 3.4.1

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Update to Decanter v7.2 which uses TW v3.4.1
- Update children: variant to use the TW out of the box *: which does the same thing (direct children selector)

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at the preview and confirm that things stay the same

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "@types/react": "^18.2",
     "@types/react-dom": "^18.2",
     "autoprefixer": "^10.4.16",
-    "decanter": "^7.1.2",
+    "decanter": "github:SU-SWS/decanter#tw-3.4.1",
```
Comment (yvonnetangsu) [inline package.json:57]: I'll replace with latest version of decanter once this branch is merged.

File: components/Banner/Banner.styles.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export const wrapper = 'mr-0 au-ml-auto flex-col lg:flex-row';
 export const contentWrapper = 'lg:rs-pr-9 ml-0';
 export const heading = '2xl:whitespace-pre-line -mt-01em rs-mb-2 xl:max-w-1200';
-export const body = 'max-w-[50ch] rs-mb-3 children:children:leading-snug';
+export const body = 'max-w-[50ch] rs-mb-3 *:*:leading-snug';
```
Comment (sherakama) [inline components/Banner/Banner.styles.ts:4]: 😨 Double star?

File: components/ChangemakerCard/ChangemakerCard.styles.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ export const imageWrapper = 'overflow-hidden aspect-w-1 aspect-h-2';
 export const info = 'rs-px-1 pb-150 absolute w-full h-full bottom-0 left-0 mb-0';
 export const heading = 'mb-02em mt-auto';
 
-export const cardContent = 'absolute w-full h-full top-0 left-0 px-20 py-30 3xl:py-48 3xl:px-36 aria-hidden:opacity-0 opacity-100 backdrop-blur-sm transition-opacity duration-500 bg-gradient-to-b from-gc-black/60 to-gc-black/90 gc-changemaker children:children:children:!mb-1em';
+export const cardContent = 'absolute w-full h-full top-0 left-0 px-20 py-30 3xl:py-48 3xl:px-36 aria-hidden:opacity-0 opacity-100 backdrop-blur-sm transition-opacity duration-500 bg-gradient-to-b from-gc-black/60 to-gc-black/90 gc-changemaker *:*:*:!mb-1em';
```
Comment (sherakama) [inline components/ChangemakerCard/ChangemakerCard.styles.ts:9]: Triple star!

**Comment by sherakama** [comment]
I rate this PR 10 *s

**Comment by yvonnetangsu** [comment]
> I rate this PR 10 *s

lol thanks for reviewing 😂 ! Want to approve the Decanter PR also? I used the minor version tag because the TW upgrade was also a minor version.
https://github.com/SU-SWS/decanter/pull/942

File: components/ChangemakerCard/ChangemakerCard.styles.ts (hunk 3 above)
Comment (yvonnetangsu) [reply components/ChangemakerCard/ChangemakerCard.styles.ts:9]: To get to those WYSIWYG elements 😂

**Comment by yvonnetangsu** [comment]
I'm going to push this through since you already reviewed @sherakama 🤠

**Pull request merged**